packages/typography/src/FormElements.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const Mode = {
+  Light: 'light',
+  Dark: 'dark',
+} as const;
+
+type Mode = typeof Mode[keyof typeof Mode];
+
+interface ColorSets {
+  labelColor: string;
+  descriptionColor: string;
+}
+
+const colorSets: Record<Mode, ColorSets> = {
+  [Mode.Light]: {
+    labelColor: css`
+      color: ${uiColors.gray.dark2};
+    `,
+    descriptionColor: css`
+      color: ${uiColors.gray.dark1};
+    `,
+  },
+  [Mode.Dark]: {
+    labelColor: css`
+      color: ${uiColors.white};
+    `,
+    descriptionColor: css`
+      color: ${uiColors.gray.light1};
+    `,
+  },
+};
+
+const labelStyle = css`
+  font-size: 14px;
+  font-weight: bold;
+  line-height: 16px;
+  padding-bottom: 4px;
+`;
+
+type LabelProps = JSX.IntrinsicElements['label'] & {
+  darkMode?: boolean;
+  id: string | number;
```

Comment:
I would call this `inputId` so it doesn't collide with the `id` of the label element itself. should we omit the `htmlFor` property if we're going to rename it? is there a good case for renaming it in the first place?

Reply:
Nope, good call... will omit id and just spread htmlFor in rest. Just want to make sure a value for the prop is supplied, so will be sure to type appropriately